libs/ui/src/screen.tsx

```diff
@@ -1,22 +1,17 @@
 import styled from 'styled-components';
 
 interface Props {
-  flexDirection?: 'row' | 'row-reverse' | 'column' | 'column-reverse';
-  voterMode?: boolean;
   white?: boolean;
+  navLeft?: boolean;
+  navRight?: boolean;
```

Comment:
Since these are mutually exclusive, should it be something like `nav="left"` or `nav="right"` instead?

```suggestion
  nav?: 'left' | 'right'
```

Reply:
You're probably right. There's something I also like about the simple boolean nature of these. But I'll probably change it.